hardware/src/support.rs

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+//! Tools for identifying supported hardware.
```

Comment:
It is a pain that these Ids need to be hard-coded, but I guess we have no alternative....
Using a file would not help here, right?

Reply:
It really wouldn't.  Even the pci database is sketchy